static/js/components/OrderSummary.js

```diff
@@ -36,20 +47,41 @@ class OrderSummary extends React.Component {
     return <p className="payment-explanation">{text}</p>;
   }
 
+  showAmount(description: string, amount: string): Array<React$Element<*>> {
+    return [
+      <Cell col={8} className="description mdl-cell--4-col-tablet mdl-cell--2-col-phone" key={description}>
```

Comment:
It looks like you can set the tablet and phone cell counts using props. From [here](https://react-mdl.github.io/react-mdl/components/grid/):

        <Cell col={6} tablet={8}>6 (8 tablet)</Cell>
        <Cell col={4} tablet={6}>4 (6 tablet)</Cell>
        <Cell col={2} phone={4}>2 (4 phone)</Cell>